### ISSUE TITLE
Any plans for realtime? Or some feedback as to how this would go?

### ISSUE THREAD

**Issue by EvoPulseGaming** [awinkler/ue4_towr#1]
I saw this a few weeks ago, and thought maybe you plan on adding more, but I don't see anything yet. So I'd like to know if you are already planning this, or if you could give me a few ideas, or point out points of interest in the code (this repo or the towr repo)

**Comment by awinkler** [comment]
No, no plans for that unfortunately. To get this real-time, you would have to call the towr function from C++ in Unreal (so somehow link these external libraries in the unreal build) and also convert the Unreal Terrain into a height map. Those two would be the critical tasks I assume. Hope that gives you a rough idea👍
